File: mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int i, a, p, j;
    do
    {
        i = get_int("Enter Height:");
    }
    while (i < 1 || i > 8);

    for (a = 0 ; a < i ; a++)
    {
        //inserting spaces to create space
        for (p = i - a ; p > 1 ; p--)
        {
            printf(" ");
        }
        {
            //insering hashes incrementally on vertical axis
            for (j = a; j >= 0; j--)
            {
                printf("#");

            }
            {
                printf("  ");
                for (j = a; j >= 0; j--)
            {
                printf("#");

            }
            {
                printf("\n");
            }
            }
        }
    }
    {}
}

File: mario.py
from cs50 import get_int

while True:
    p = get_int("size of pyramid: ")
    if p >= 1 and p<=8:
      break

y = 1
for i in range(p):
    x= p-1
    print(" "*x,"#"*y, end="\n")
    y = y+1
    p = p-1
